src/components/side_panel/chart/scorecard_chart_panel/scorecard_chart_design_panel.xml

```diff
@@ -16,6 +16,16 @@
             onChange.bind="updateHumanizeNumbers"
           />
         </Section>
+        <Section class="'pt-0'">
```

Comment:
Woudln't it be better to have a font size selector (like the one in the topbar) next to the other font options of the title (bold, italic, align ....) ?